src/app/report-rollup/rollup-summary-pie-chart/rollup-summary-pie-chart.component.ts

```diff
@@ -50,7 +50,14 @@ export class RollupSummaryPieChartComponent implements OnInit {
       valuesArr = this.pieChartData.map(dataItem => {
         return dataItem.energyUsed
       });
-      textTemplate = '%{label}:<br>%{value:,.0f} ' + this.energyUnit;
+      //Chnage here
+      let newEnergyUnit: string;
+      if(this.energyUnit == 'hp'){
```

Comment:
Similar situation to rollup-summary-energy-table... if you look at lines 16/17 here energyUnit is being passed as @Input from the parent(psat-rollup.component/fsat-rollup-component). It may be better to create an energyUnit property on the parent and just pass it in here.